### PR TITLE
updates per PR5010

### PR DESCRIPTION
**Pull request by peiying2**
amendments to the guidelines per bug 1364268

### CODE REVIEW

File: en-US/mozorg/about/governance/policies/participation.lang [hunk 1]
```diff
@@ -386,5 +386,13 @@ This set of guidelines is distributed under a <a href="%(license)s" rel="license
...
+Mozilla may amend the guidelines from time to time and may also vary the procedures it sets out where appropriate in a particular case. Your agreement to comply with the guidelines will be deemed agreement to any changes to it. In case Mozilla amends the guidelines, the updated version can be found at <a href="{{ guidelines }}">https://www.mozilla.org/about/governance/policies/participation/</a>. You can sign up for push updates by subscribing to <a href="{{ group }}">https://groups.google.com/d/forum/mozilla-cpg</a>. This policy does not form part of any Mozilla employee’s contract of employment or otherwise have contractual effect.
```
Comment (flodolo) [inline en-US/mozorg/about/governance/policies/participation.lang]: This string won't work. Take a look at how this variable looks like in the template and in the lang file
https://github.com/craigcook/bedrock/blob/0c5719c06aad955f4effc4d8c2fb3b7152cfb29c/bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html#L508

Is there a reason to do this separately from other pending work?

Comment (peiying2) [inline en-US/mozorg/about/governance/policies/participation.lang]: Should I change it to:
<a href="%(guidelines)s">https://www.mozilla.org/about/governance/policies/participation/</a>

for the 2nd question, I thought this was easier so I want to get it out of the way plus the two opt-in bugs, and work on the rest together later.

Comment (peiying2) [inline en-US/mozorg/about/governance/policies/participation.lang]: you have to check on the comment in edit view, somehow it is doesn't show more than the link itself.

Comment (flodolo) [inline en-US/mozorg/about/governance/policies/participation.lang]: I sad look at the variable ;-)
`<a href="{{ rustlang_coc }}">` in the template becomes `<a href="%(rustlang_coc)s">`in the lang file.

Comment (peiying2) [inline en-US/mozorg/about/governance/policies/participation.lang]: so the change is from <a href="{{ group }}"> to <a href="%(guidelines)s">, correct?

Comment (flodolo) [inline en-US/mozorg/about/governance/policies/participation.lang]: yes

**Pull request merged**